waiter/src/waiter/core.clj

```diff
@@ -107,7 +107,7 @@
                                      :async-result-handler-fn
                                      ["/status/" :request-id "/" :router-id "/" :service-id "/" :host "/" :port "/" [#".+" :location]]
                                      :async-status-handler-fn}
-                     "waiter-auth" :waiter-auth-handler-fn
+                     "waiter-auth" {true :waiter-auth-handler-fn}
```

Comment:
So we'd have 
<pre>
{"" :waiter-auth-handler-fn
 "/extensions" :waiter-auth-handler-fn}
</pre>
?

Reply:
It seems good to me to allow the auth handler full control over the `/waiter-auth/*` path.

Reply:
I was thinking more along the lines of `"/extensions" :waiter-auth-extensions-handler-fn`.

Reply:
Bind it to a no-op if there are no extensions?  I'm having trouble seeing the benefit.

Reply:
As I mentioned earlier I wanted to avoid the handler counter clashes. I guess we can just have a single counter for all `/waiter-auth...` requests and then other counters inside oauth for the different endpoints.